.github/workflows/release-branch.yml

```diff
@@ -123,12 +123,21 @@ jobs:
         with:
           dry-run-flag: ${{ inputs.dry-run-flag }}
       - uses: ./.github/actions/download-node-modules-and-artifacts
-      - name: build libraries
+      - name: Set libraries versions
         run: |
           set -u;
           ./scripts/update-version.sh -gnu || exit 1;
+      - name: Set migrations
```

Comment:
Here we are setting migration version to e.g. 7.9.0-1234567890 (current release version) in `lib/core/schematics/migrations/collection.json`